### ISSUE TITLE
Accessibility: Zoom in does not work

### ISSUE THREAD

**Issue by jithinjosepkl** [wixtoolset/issues#5999]
* Which version of WiX are you building with?

3.10.3.3010

* Which version of Visual Studio are you building with (if any)?

VS 2017

* Which version of .NET are you building with?

4.0

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

Windows 10

* Describe the problem and the steps to reproduce it.

Zoom in is not supported with Ctrl +. Attaching video from accessibility team 
[3765620_A11y_MSMPI_MSMPISDKSetupScreen_Zoom.zip](https://github.com/wixtoolset/issues/files/3140021/3765620_A11y_MSMPI_MSMPISDKSetupScreen_Zoom.zip)
for more clarity.

**Comment by barnson** [comment]
Windows Installer is in charge of the UI at runtime. WiX is the build tool and cannot influence runtime behavior except as MSI allows. MSI would need to implement a change like this.
